multimedia/ffmpeg4/Makefile

```diff
@@ -0,0 +1,342 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ffmpeg4
+PKG_VERSION:=4.1.4
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=
+PKG_LICENSE:=GPL-2.0-or-later LGPL-2.1-or-later LGPL-3.0-only
+PKG_LICENSE_FILES:=COPYING.GPLv2 COPYING.GPLv3 COPYING.LGPLv2.1 COPYING.LGPLv3
+
+PKG_SOURCE:=ffmpeg-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.ffmpeg.org/releases/
+PKG_HASH:=f1f049a82fcfbf156564e73a3935d7e750891fab2abf302e735104fd4050a7e1
+PKG_BUILD_DIR:=$(BUILD_DIR)/ffmpeg-$(PKG_VERSION)
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
```

Comment:
Should use PKG_USE_MIPS16:=0 unconditionally.

Reply:
Last time I messed with it the compiler barfed when mixing MIPS16 and non-MIPS16 libraries (world/base). I have little interest to revisit it again so unless you want to test it I'm going to say no for now.